backend/service/auditsink/slack/slack_helper.go

```diff
@@ -0,0 +1,28 @@
+package slack
+
+import (
+	auditv1 "github.com/lyft/clutch/backend/api/audit/v1"
+	configv1 "github.com/lyft/clutch/backend/api/config/service/auditsink/slack/v1"
+	"github.com/lyft/clutch/backend/middleware"
+)
+
+// GetSlackOverrideText returns the custom slack message from the slack override config if the FullMethod matches
+// the serive and method in the audit event. Otherwise ok is false and an empty string is returned.
+func GetSlackOverrideText(override *configv1.Override, event *auditv1.RequestEvent) (string, bool) {
```

Comment:
nit in general try to avoid repeating the package name in its members
```suggestion
func GetOverrideText(override *configv1.Override, event *auditv1.RequestEvent) (string, bool) {
```

Reply:
also method can be private

Reply:
wanted to keep it public so that we could use it in the internal implementation

Reply:
discussing in other comment thread https://github.com/lyft/clutch/pull/1089#discussion_r586589334